bridges/otelzap/encoder.go

```diff
@@ -201,17 +201,24 @@ func (a *arrayEncoder) AppendString(v string) {
 	a.elems = append(a.elems, log.StringValue(v))
 }
 
-// TODO.
-func (a *arrayEncoder) AppendComplex128(v complex128)  {}
-func (a *arrayEncoder) AppendUint64(v uint64)          {}
-func (a *arrayEncoder) AppendComplex64(v complex64)    {}
-func (a *arrayEncoder) AppendDuration(v time.Duration) {}
-func (a *arrayEncoder) AppendInt32(v int32)            {}
-func (a *arrayEncoder) AppendInt16(v int16)            {}
-func (a *arrayEncoder) AppendInt8(v int8)              {}
-func (a *arrayEncoder) AppendTime(v time.Time)         {}
-func (a *arrayEncoder) AppendUint(v uint)              {}
-func (a *arrayEncoder) AppendUint32(v uint32)          {}
-func (a *arrayEncoder) AppendUint16(v uint16)          {}
-func (a *arrayEncoder) AppendUint8(v uint8)            {}
-func (a *arrayEncoder) AppendUintptr(v uintptr)        {}
+func (a *arrayEncoder) AppendComplex128(v complex128) {
+	r := log.Float64("r", real(v))
+	i := log.Float64("i", imag(v))
+	a.elems = append(a.elems, log.MapValue(r, i))
```

Comment:
I like this approach, it allows downstream to restructure without, say, having to parse a string (i.e. `"1+i3"`).

Can we use more descriptive names here though?

```suggestion
	r := log.Float64("real", real(v))
	i := log.Float64("imag", imag(v))
	a.elems = append(a.elems, log.MapValue(r, i))
```

Reply:
I suggest doing it on separate PR as this is how objectEncoder already works.

@MrAlias are you fine with it?